ui/components/component-library/label/README.mdx

```diff
@@ -1,34 +1,38 @@
 import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
 
+import { Text } from '..';
+
 import { Label } from './label';
 
 # Label
 
-The `Label` is a component used to label form inputs.
+The `Label` is a component used to label form inputs
```

Comment:
Not using periods at the end of single line sentences we should adhere to our content design stye guide when possible

![Screenshot 2022-11-24 at 11 55 27 AM](https://user-images.githubusercontent.com/8112138/203856767-6c6c3a7b-74f7-413f-9703-47e070c5dc40.png)